util/minetestmapper.py

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python2
+# expertmm fork from 2016-02-01 git version
```

Comment:
You should put the attribution to you to where the other people are attributed. There is a list of changes, add yours to it.